cluster/gce/gci/configure-helper.sh

```diff
@@ -752,10 +752,10 @@ function start-kube-apiserver {
   fi
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
 
-  if [[ -n "${KUBE_USER:-}" ]]; then
+  if [[ -n "${KUBE_USER}" ]]; then
```

Comment:
No, `"${KUBE_USER:-}"` doesn't set KUBE_USER. Use `${KUBE_USER:=admin}` if you want to initialize it.